tests/quantization/test_onnx_export.py

```diff
@@ -52,6 +52,7 @@ def test_onnx_export_to_fake_quantize(tmp_path):
 def test_onnx_export_to_quantize_dequantize(tmp_path):
     model = TwoConvTestModel()
     nncf_config = get_config_for_export_mode(should_be_onnx_standard=True)
+    nncf_config['quantizer_setup_type'] = 'pattern_based'
```

Comment:
Should test the default method here; would be even better if we tested both.

Reply:
Done.